specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/Patches.json

```diff
@@ -0,0 +1,524 @@
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Patches_Delete_0": {
+            "$ref": "./examples/Patches_Delete.json"
+          }
+        }
+      }
+    },
+    "/subscription/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/builders/{builderName}/patches/{patchName}/skip/{patchSkipConfig}": {
+      "post": {
+        "tags": [
+          "Patches"
+        ],
+        "operationId": "Patches_Skip_Configure",
+        "description": "Configure the patch skip option by patch name.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./Builders.json#/parameters/BuilderNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PatchNameParameter"
+          },
+          {
+            "name": "patchSkipConfig",
+            "in": "path",
+            "description": "Set the patch to be skipped or not.",
+            "type": "boolean",
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Patch skipping detail updated successfully."
+          },
+          "202": {
+            "description": "Updating patch skipping detail.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Patches_Skip_Configure_0": {
+            "$ref": "./examples/Patches_Skip_Configure.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/builders/{builderName}/patches/{patchName}/apply": {
+      "post": {
```

Comment:
can this be a create operation on patch?

Reply:
Instead of created by user, patch is generated by the system. Therefore, the `POST` request here is only doing the action of applying the patch.

Reply:
This means its impossible for people to declaratively author a list of whatever patches they want applied, with e.g. ARM templates - is that what you want? @harryli0108

Reply:
Yes. Because of the patches are generated for container app revision, and currently we don't support batch applying patches, patches can only be applied by specifying patch name here.